README.md

```diff
@@ -1,20 +1,46 @@
-# Hackweek Jupyterbook Template Repository
+# Hackweek Website Template
```

Comment:
should we also change the repo's name?

Reply:
haha, perhaps! i'll open an issue for discussion

Reply:
https://github.com/uwhackweek/jupyterbook-template/issues/69